examples/README.md

```diff
@@ -4,15 +4,20 @@ Here you can find notebooks showing features and example usage of pytket.
 
 *Note: When viewing our examples, github has a bug where it occasionally can't render Jupyter notebooks properly. To bypass this, you can either clone the repo and open the examples in Jupyter yourself (this is recommended! You can play around with them), or you can enter the corresponding URL into https://nbviewer.jupyter.org and view examples there.*
 
-[cirq_routing_example](https://github.com/CQCL/pytket/blob/master/examples/cirq_routing_example.ipynb) - Cirq interface and routing.
+[backends_example](https://github.com/CQCL/pytket/blob/master/examples/backends_example.ipynb) - Running circuits on our allowed backends.
 
+[circuit_generation_example](https://github.com/CQCL/pytket/blob/master/examples/circuit_generation_example.ipynb) - Advance methods for circuit generation in pytket.
 
-[pyquil_example](https://github.com/CQCL/pytket/blob/master/examples/cirq_routing_example.ipynb) - PyQuil interface and leveraging features across platforms.
+[circuit_analysis_example](https://github.com/CQCL/pytket/blob/master/examples/circuit_analysis_example.ipynb) - Methods for visualizing and anaylsing circuits in pytket.
```

Comment:
Elsewhere in our docs we quite consistently use trad. British "ise" spellings. Although I personally prefer "ize" spellings, we should be consistent -- so, "visualising" here.